### DISCUSSION TITLE
Create a Layer 2 Fabric with AVD?

### DISCUSSION THREAD

**Discussion by roberthill-omf** [aristanetworks/avd#1498] [Q&A] [answered]
My organization is building a new datacenter fabric but don't require the size and scalability of a layer 3 evpn/vxlan fabric. Is there a way to utilize AVD to build out and maintain a smaller layer 2 fabric? I'm new to network automation and Ansible so I apologize if I've overlooked an easy answer or solution.

**Comment by ClausHolbechArista** [comment, marked as answer]
Hi Robert,
We are working on getting a "turn key" L2LS design included into AVD. Today it would require many tweaks to get it working, but it can be done.
What are your timelines for this project?

**Reply by roberthill-omf** [reply]
Thank you for your quick response. 

We have two datacenters we'll be migrating. The first one will be within the next three months. The second will probably be started in Q4 of this year. The planning is already underway for the first design and our engineers are planning on building and maintaining the containers and configlets by hand. 

After researching and tooling around with the examples offered for AVD, I was hoping to be able to utilize it for our projects. If not the first build out, then hopefully the second. We as a team are absolute novices when it come to Ansible and while I'm sure we'd be able to cobble something together eventually to meet our needs, having an OEM and community backed solution would be better for us in the long run.

**Reply by ClausHolbechArista** [reply]
I think we can have L2LS support in AVD before then. At least before the second roll-out, but maybe even in time for the first.

I would suggest that you work with your account team from Arista, to get help from our Professional Services team to get started with AVD. 

PS can help you get a jump start and they are excellent in helping customers getting started on the automation journey. The PS team can also assist with developing any missing things in the AVD collection, so you don't get stuck on some missing feature.

**Reply by roberthill-omf** [reply]
Awesome, thank you for the suggestion. I will reach out to our ambassador about getting PS involved. Thank you for the help.
